src/librawspeed/decoders/Cr2Decoder.cpp

```diff
@@ -273,11 +273,10 @@ RawImage Cr2Decoder::decodeNewFormat() {
 
 RawImage Cr2Decoder::decodeRawInternal() {
   try {
-    if (hints.find("old_format") != hints.end()) {
+    if (hints.find("old_format") != hints.end())
       return decodeOldFormat();
-    } else {
-      return decodeNewFormat();
-    }
+
+    return decodeNewFormat();
```

Comment:
Hmm, but it helps to write spaghetti-like code, code with unneeded `{}`
In **this** `Cr2Decoder::decodeRawInternal()` case i agree,
but in general, i'm not so sure it wouldn't be be a positive to discourage it...